src/Promise.fs

```diff
@@ -64,6 +64,9 @@ let tryStart (fail: Exception->unit) (pr: JS.Promise<'T>): unit = jsNative
 [<Emit("Promise.all($0)")>]
 let Parallel (pr: seq<JS.Promise<'T>>): JS.Promise<'T[]> = jsNative
 
+[<Emit("Promise.all($0)")>]
+let all (pr: seq<JS.Promise<'T>>): JS.Promise<'T[]> = jsNative
```

Comment:
Actually I like more your approach in the other PR, just calling `JS.Promise.all` instead of abusing `Emit`. Maybe we should do that with the other helpers that call `Promise` static methods, what do you think?

Reply:
Ah yes, you are right :) 

I created the PR using Github interface so I didn't know about `JS.Promise.all` :D

But this will need a change in [Fable.Core.JS](https://github.com/fable-compiler/Fable/blob/master/src/Fable.Core/Fable.Core.JS.fs#L213)

`abstract all: [<ParamArray>] values: obj[] -> Promise<obj>` should returns `Promise<obj seq>`

Reply:
Ah, you're right. Ok, let's merge it for now and we'll change it together with the other cases later. Thanks again! :+1: